src/data/data_entry_flow.ts

```diff
@@ -58,8 +58,18 @@ export interface DataEntryFlowStepAbort {
   description_placeholders: { [key: string]: string };
 }
 
+export interface DataEntryFlowStepProgress {
+  type: "progress";
+  flow_id: string;
+  handler: string;
+  step_id: string;
+  progress_action: string;
+  description_placeholders: { [key: string]: string };
```

Comment:
We should bring the general fields to a generic type someday, they all have `flow_id`, `handler`, `step_id` and `description_placeholders`